voice/voice-ai/x/SpeechRecognition/SpeechRecognition.swift

```diff
@@ -317,10 +318,15 @@ class SpeechRecognition: NSObject, ObservableObject, SpeechRecognitionProtocol {
             guard !response.isEmpty else {
                 return
             }
+            
+            let startTime = Int64(Date().timeIntervalSince1970 * 1000000)
             registerTTS()
             
             if !isRepeatingCurrentSession {
                 textToSpeechConverter.convertTextToSpeech(text: response)
+                
+                let endTime = Int64(Date().timeIntervalSince1970 * 1000000)
```

Comment:
.convertTextToSpeech enqueues the text for processing, so this is not really measuring TTS or the delay until playback